iquip/apps/logger.py

```diff
@@ -101,12 +98,12 @@ def __init__(self, parent: Optional[QObject] = None):
         layout.addWidget(self.buttonBox)
 
     def buttonOKClicked(self):
-        """Clicks OK to clear log."""
+        """Called when the OK button is clicked."""
         self.confirmed.emit()
         self.close()
 
     def buttonCancelClicked(self):
-        """Clicks Cancel not to clear log."""
+        """Clicked Cancel not to clear log."""
```

Comment:
For consistency, please follow the form of `buttonOKClicked()` docstring.
"Called when the Cancel button is clicked."

Reply:
I had fixed it in this way, but since it is not requested part, I just moved to the original one.
I will change it.